content/home/examples/a-stateful-component.md

```diff
@@ -1,7 +1,7 @@
 ---
-title: A Stateful Component
+title: En komponent med state
```

Comment:
I'm unsure what approach to take here, maybe Stateful would be best translated as 'med state' while Stateless would be 'utan state'. What I'm most conflicted about is that there exists a pretty good translation for Stateless which is 'tillståndslös'.

Another approach would be to call Stateful and Stateless proper nouns and don't translate them at all.

What do you think?

Reply:
@rloqvist  As I think I've mentioned somewhere, my approach has been to use the same type of language that I use when I speak to my peers, and I've never used the word "tillståndslös" or "tillståndsfull" (does that exist?). "state" and "props" seem to have become common enough between Swedish React developers at least.

On the other hand I do not have any issue with the suggestion of using "proper nouns" (although I think it might be more adjective?). I have had occurrences where I've used those words when speaking Swedish. "En Stateful komponent" and "En Stateless komponent" works for me 👍

Reply:
@rloqvist While translating "Tutorial" page there are several instances of the word "state" that mean the noun, not the adjective (as above). So "En stateful komponent" does work, but it does not address the translation of
>To "remember" things, components use **state**.
Which I translated as
>För att "komma ihåg" saker använder komponenter sig av **state**.

Maybe this "noun form" is not the same discussion as this threads "adjective form"?